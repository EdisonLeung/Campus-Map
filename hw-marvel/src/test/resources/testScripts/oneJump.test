# Tests a simple pathfinding with one jump from one node to another

# Create Graph
CreateGraph oneJump

# Add nodes
AddNode oneJump a
AddNode oneJump b

# Add edge
AddEdge oneJump a b AB

# Find path
FindPath oneJump a b