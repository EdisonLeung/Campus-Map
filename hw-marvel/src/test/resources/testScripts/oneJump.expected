# Tests a simple pathfinding with one jump from one node to another

# Create Graph
created graph oneJump

# Add nodes
added node a to oneJump
added node b to oneJump

# Add edge
added edge AB from a to b in oneJump

# Find path
path from a to b:
a to b via AB